packages/fluentui/react/src/themes/teams/components/Input/inputStyles.ts

```diff
@@ -42,6 +42,15 @@ const inputStyles: ComponentSlotStylesPrepared<InputProps & InputState, InputVar
       opacity: 1 // undo Firefox default opacity
     },
 
+    // Overrides for "disabled" inputs
+    ...(p.disabled && {
+      color: v.colorDisabled,
+      boxShadow: 'none',
+      '::placeholder': {
+        color: v.colorDisabled
+      }
```

Comment:
```suggestion
```

This will overwrite previous rule completely I suggest to change it to:

```
    '::placeholder': {
      color: v.placeholderColor,
      opacity: 1 // undo Firefox default opacity,
      ...p.disabled && { color: v.colorDisabled },
    },
```

Reply:
changed, thanks!